### ISSUE TITLE
Section 3.1: the accessibility of communications should be distinguished from communication about accessibility.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/maturity-model#160]
Section 3.1 addresses the accessibility of communications , but also seeks evidence of communication (e.g., published conformance documentation) about digital accessibility in general.

I would suggest more clearly specifying and distinguishing these two issues in the introduction of the section.  Section 3.1.1.2 could be subdivided, addressing accessibility of communications in 3.1.1.2, and then inserting a new section (3.1.1.3, renumbering accordingly) on communications about digital accessibility.

**Comment by swinehartganderson** [comment]
Hi @jasonjgw, can you provide more details or examples of what you think needs to be differentiated to help clarify?

**Comment by jasonjgw** [comment]
Good examples of this in the current draft (section 3.1.1.2) include: accessibility statements, statement of commitment to accessibility, accessibility compliance documentation, and accessibility conformance reports, all of which are communication about accessibility, but they are mixed in with requirements for making communications themselves accessible. My suggestion is to separate these out, so that requirements regarding the accessibility of communication in general are dealt with in one section, and requirements for providing information/communication about accessibility are addressed in another section. See my original comment for more explanation.

**Comment by Helixopp** [comment]
Please review the latest editors draft, as of 29 August 2024. We have revised this Dimension, and believe it addresses this issue. Please reopen this issue if you disagree.
